lab_5_scrapper/scrapper.py

```diff
@@ -168,6 +295,15 @@ def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        text = article_soup.find_all(itemprop="articleBody")
+        description = article_soup.find(itemprop="description")
+        article = ''
+        if description:
+            article += description.text
+        for paragraph in text:
+            article += paragraph.text
```

Comment:
concatenation is bad. use better way for building one string from many small ones

Reply:
Done